nni/retiarii/serializer.py

```diff
@@ -149,3 +151,15 @@ def basic_unit(cls):
     import torch.nn as nn
     assert issubclass(cls, nn.Module), 'When using @basic_unit, the class must be a subclass of nn.Module.'
     return serialize_cls(cls)
+
+
+def model_wrapper(cls):
```

Comment:
it is not clear how to use this wrapper. what is the difference between `model_wrapper` and `serializer`?

Reply:
I think it's documented in the docstring.